lib/FilesHooks.php

```diff
@@ -229,7 +229,9 @@ protected function addNotificationsForFileAction($filePath, $activityType, $subj
 
 		$accessList = $this->getUserPathsFromPath($filePath, $uidOwner);
 
-		$this->generateRemoteActivity($accessList['remotes'], $activityType, time(), $this->currentUser->getCloudId(), $accessList['ownerPath']);
+		if (!empty($accessList)) {
```

Comment:
```suggestion
		if (!empty($accessList['remotes'])) {
```

?

Reply:
Arg, my bad, thanks for catching :see_no_evil: